sdk/python/feast/infra/offline_stores/bigquery.py

```diff
@@ -571,27 +573,16 @@ def to_remote_storage(self) -> List[str]:
                 "offline store when executing `to_remote_storage()`"
             )
 
-        table = self.to_bigquery()
+        assert isinstance(self.config.offline_store, BigQueryOfflineStoreConfig)
```

Comment:
is this for type checking? Why wasn't this needed before?

Reply:
Yes and getting autocompletion from IDE with field names.